queries/runescript/highlights.scm

```diff
@@ -0,0 +1,100 @@
+; Variables
+[
+  (identifier)
+  (local_variable)
+  (game_variable)
+  (constant_variable)
+  (parameter_variable)
+  (declaration_variable)
+] @variable
+
+; Scripts
+(script
+  trigger: (identifier) @keyword.directive
+  subject: (identifier) @function)
+
+; Calls
+(call
+  [
+    "~"
+    "@"
+  ]? @function
```

Comment:
Seems like these should be `@operator`? Or am I wrong in this

Reply:
Same as above; those prefixes define the type of a call `~call()` vs `@call()` vs `call()`. I kept them the same color as the callee itself, but if you think they should be highlighted differently lmk.

Reply:
Type in what sense?

Reply:
"~some_proc()" calls a procedure script (something that returns) and "@some_label()" jumps to a label (a script that doesn't return). I think in this case we can go with the way you proposed in the comment above.

Reply:
That is... special :) Hard to find a good bucket for that -- maybe `@attribute`(`.builtin`)? Otherwise `@punctuation.special` could work.

Basically, my thinking is this: if the grammar does not include these as part of the "function" node, they should not be treated as part of the function name but captured separately. _What_ they should be captured as is another question, which can be hard to answer for non-standard languages. Here we go with best-effort and more leeway to language maintainers.

Reply:
> That is... special :)

Brought to us by Andrew Gower himself at the dawn of the century :)

> maybe @attribute(.builtin)?

This one actually fits pretty well.